examples/rl/ddpg_pendulum.py

```diff
@@ -414,14 +435,17 @@ def policy(state, noise_object):
 target_actor.save_weights("pendulum_target_actor.h5")
 target_critic.save_weights("pendulum_target_critic.h5")
 
+
 """
 Before Training:
 
 ![before_img](https://i.imgur.com/ox6b9rC.gif)
+
 """
 
 """
 After 100 episodes:
 
 ![after_img](https://i.imgur.com/eEH8Cz6.gif)
+
```

Comment:
How did the extra whitespace get added? You can remove it.

Reply:
Yeah, Many blank lines are added by `black`.

Reply:
`black` does not add any such blank lines. Let's get rid of them.

- Start from the original file (on the `master` branch)
- Add the `training=True` statements and modify the "last modified" date
- Run `black ddpg_pendulum.py`
- Commit the file